src/status_im2/contexts/share/style.cljs

```diff
@@ -0,0 +1,97 @@
+(ns status-im2.contexts.share.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def screen-padding 20)
+
+(def blur
+  {:style        {:position :absolute :top 0 :left 0 :right 0 :bottom 0}
+   :overlayColor colors/neutral-80-opa-80
```

Comment:
This can be kebab case.

Reply:
fixed here : https://github.com/status-im/status-mobile/pull/15528/commits/5196d3e55f4e4110edd484d70242c12d9e1adeb8